.pre-commit-config.yaml

```
@@ -0,0 +1,6 @@
+repos:
+-   repo: git://github.com/doublify/pre-commit-clang-format
+    rev: master
```

Per https://pre-commit.com/#using-the-latest-version-for-a-repository
This needs to be a specific commit.

Change `master` to `62302476d0da01515660132d76902359bed0f782`

---

Nope - usually a tag is used rather than a hash

---

ah just realised they don't have tags in their repo, yup hash it is...

```suggestion
   rev: '62302476'
```

---

I haven't got my laptop currently to confirm it works with the string and partial hash, but can't see why it wouldn't